src/caffe/proto/caffe.proto

```diff
@@ -154,7 +154,7 @@ message SolverParameter {
   optional string lr_policy = 8; // The learning rate decay policy.
   optional float gamma = 9; // The parameter to compute the learning rate.
   optional float power = 10; // The parameter to compute the learning rate.
-  optional float momentum = 11; // The momentum value.
+  optional float momentum = 11 [default = 0.9]; // The momentum value.
```

Comment:
@shelhamer does a default momentum of 0.9 break other things in caffe? @PatWie it might be easier to have that default be handled in the ADAM layer if desired.